### DISCUSSION TITLE
Can I concatenate multiple fields?

### DISCUSSION THREAD

**Discussion by MitchellAJ** [coddingtonbear/obsidian-csv-table#20] [General]
I have a table with surname and firstname, I would like to display them as:

Smith, John

using the expression option.  I tried + . etc, but no lunk.

Any advice?

**Comment by karlfast** [comment]
I have also been unable to concatenate strings.

My use case is one column with `{text}`, another with `{url}`.  I want concatenate them to create a markdown link `[{text}]({url})`.

I'd render the link in a new column, then filter out the '{text}` and `{url}` columns.

Tried using `+` and `and` operators with no success.
